exercises/space-age/description.md

```diff
@@ -1,4 +1,4 @@
-Given an age in seconds, calculate how old someone would be on:
+Given a person's age in seconds, calculate their age in terms of years of the different planets.
```

Comment:
> a person's 

That it's a person is largely irrelevant.

> years of the different planets

This does not make grammatical sense to me.


Disclaimer: I don't think there is anything wrong with the original wording.

Reply:
> That it's a person is largely irrelevant.

True, but supplying an object for the noun makes it easier for students to understand by adding a little more context. Of course _we_ know what it means as-is, but people that don't already know may struggle - as evidenced by the fact that the issue was raised.

>>years of the different planets
>
>This does not make grammatical sense to me.

Maybe this should be "years _on_ the different planets"? Or maybe even "according to the length of a year on a different planet"